docs/configuration.md

```diff
@@ -518,8 +518,26 @@ To disable removing frame-ancestors from Content-Security-Policy and X-Frame-Opt
 const configuration = { features: { sessionManagement: { keepHeaders: true } } };
 ```
 
+To calculate session state, oidc-provider uses the client's host (extracted from
```

Comment:
It doesn't use the client's host extracted from the `redirect_uri`, it uses the `redirect_uri`'s Origin as defined by an RFC (see the specs)